egs/imagenet/v1/local/prepare_data.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (author: Chun-Chieh "Jonathan" Chang)
+
+# This script loads the training and test data for
+# Imagenet 2012 Task 1 or 3 Classification
+
+# Currently the script is set to run the one for Task 3
+# To change modify the paths to the correct directories
+
+[ -f ./path.sh ] && . ./path.sh;
+
+# Path to imagenet directory
+# Requires the datasets for Task 1:
+# "development kit", "training images", "validation images", and "test images"
+dl_dir=/export/b18/imagenet_2012/
+
+# Various other paths
+devkit_dir=$dl_dir/devkit_t3
```

Comment:
Everything that is extracted from the original download dir, should be done in a local directory like `data/local`.
Assume the download-dir can be read-only. Please see `prepare_data.sh` in IAM and UW3